components/Carousel.js

```diff
@@ -6,75 +6,92 @@ import Events from "./Events";
 
 const Carousel = () => {
   return (
-    <div className="horizontal-scroll-wrapper  rectangles">
-      <div>
-        <AboutNew />
-      </div>
-      <div>
-        <TapList />
-      </div>
-      <div>
-        <Events />
-      </div>
-      <div>
-        <Blog />
-      </div>
-      <div>
-        <Brews />
+    <div id="container">
+      <div id="container2">
+        <div id="about-new" className="box one">
+          <AboutNew />
+        </div>
+        <div id="tap-list" className="box two">
+          <TapList />
+        </div>
+        <div id="events" className="box three">
+          <Events />
+        </div>
+        <div id="blog" className="box four">
+          <Blog />
+        </div>
+        <div id="brews" className="box five">
+          <div>
+            <Brews />
+          </div>
+        </div>
       </div>
       <style jsx>{`
-        ::-webkit-scrollbar {
-          width: 2px;
-          height: 2px;
+        #container .box {
+          width: 100vw;
+          height: 100vh;
+          display: inline-block;
+          position: relative;
         }
-        ::-webkit-scrollbar-button {
-          width: 2px;
-          height: 2px;
+        #container .box > div {
+          /*width: 100%;
+          height: 100%;
+          color: #fff;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          margin: -50px 0 0 -50px; */
+          line-height: 0.7;
         }
-
-        div {
-          box-sizing: border-box;
+        #container {
+          overflow-y: scroll;
+          overflow-x: hidden;
+          transform: rotate(270deg) translateX(-100%);
+          transform-origin: top left;
+          background-color: #999;
+          position: absolute;
+          width: 100vh;
+          height: 100vw;
```

Comment:
I think this is over complicating things, and maybe you're just trying to do too much at once? I put together a quick example (I'll put up a PR in a minute) to show a simpler approach, mainly: rotate the nav only (rather than rotate container and then rotate it's child "back") and use flexbox and overflow to get the horizontal scroll.